ci/docker/install/centos7_install_ccache.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+# Script to build ccache for centos7 based images
+
+set -ex
+
+pushd .
+
+yum -y install epel-release
+yum -y install git
+yum -y install autoconf
+yum -y install wget
+yum -y install make
+yum -y install google-perftools
+yum -y install asciidoc
+yum -y install gcc-c++-4.8.*
```

Comment:
isn't this part of centos7_core.sh . ? we should not duplicate it I think

Reply:
This script is called sometimes separately from install core.

Reply:
installing a compiler should not be in this file

Reply:
In this file only requirements to build ccache are installed. It works after calling install core or on it's own.

Reply:
When is it called separately from install core? These scripts are not supposed to be standalone but only serve what they are intended to do.

Reply:
It is called separately for all cross compilation platforms.

Reply:
Well but the cross compilation platforms also bring their own compilers, right? I think we should really doublecheck what's actually related to ccache and what's a better fit for another place

Reply:
I really don't think that installing utilities like asciidoc, wget, git or all that stuff is actually related to ccache.

Reply:
Here are [ccache installation instructions](https://github.com/ccache/ccache/blob/master/doc/INSTALL.md).

Reply:
The compilers for cross compilation will be in a different head (different FROM)